modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PersistentPageMemoryDataRegion.java

```diff
@@ -98,13 +98,14 @@ public PersistentPageMemoryDataRegion(
      * Starts a persistent data region.
      */
     public void start() {
-        PersistentPageMemoryDataRegionView dataRegionConfigView = cfg.value();
+        // TODO: KKK why is this cast needed?
```

Comment:
?

Reply:
todo without a ticket)

Reply:
Removed, artifact of my research process.